docs/GettingStartedDocs/VisualStudioLinux.md

```diff
@@ -0,0 +1,147 @@
+  Workloads -> Other Toolsets -> Linux Development with C++
+- [Open Enclave Wizard - Preview](https://marketplace.visualstudio.com/items?itemName=MS-TCPS.OpenEnclaveSDK-VSIX)
+  Visual Studio extension, v0.5 or later.  The extension can be installed via that marketplace link, or from within
+  Visual Studio.  (For VS2017, do Tools -> Extensions and Updates -> Online -> search for "enclave".  For VS2019,
+  do Extensions -> Manage Extensions -> Online -> search for "enclave".)
+
+You will also need a build machine running Ubuntu 16.04 (64-bit) or Ubuntu 18.04.  This can be
+a remote Linux machine, or can simply be a "Generation 2" Linux VM on the Windows
+development machine.  Ideally, the machine should be SGX capable (see [here](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/SGXSupportLevel.md) for how to
+determine this), but a non-SGX machine can still be used in simulation mode.  To set up a Linux VM
+on your Windows machine, do the following:
+
+1. Download an ISO for Ubuntu [18.04](http://releases.ubuntu.com/18.04/) or [16.04](http://releases.ubuntu.com/16.04/).
+   A "Server install image" is sufficient.
+1. Create a VM as follows.  Open "Hyper-V Manager", and do Action -> New -> Virtual Machine....  
+   - On the Specify Generation screen, choose Generation 2.
+   - On the Configure Networking screen, choose Default Switch to ensure you can connect to it with a debugger.
+   - On the Installation Options screen, choose the ISO file you downloaded.
+   - All other options can be either left as the defaults or changed as desired.
+1. Disable Secure Boot as follows.  In Hyper-V Manager, right click on the VM you created while it is stopped,
+  and select Settings... -> Security and uncheck Enable Secure Boot.
+1. Uncheck "Enable checkpoints" under the VM's Settings -> Checkpoints, since SGX will not work with checkpoints.
+1. If using an SGX-capable machine, enable SGX for the VM as follows (this cannot be done from Hyper-V Manager):
+   - Download [VirtualMachineSgxSettings.psm1](https://raw.githubusercontent.com/microsoft/openenclave/f28cedce63be9673e20fe54563987189f2565637/new_platforms/scripts/VirtualMachineSgxSettings.psm1)
+   - Open an elevated PowerShell window (e.g., type "powershell" and click Run as Administrator)
+   - Invoke the following commands, using the path to where you downloaded the file, and replacing MyVM with your VM name:
+   ```
+   Set-ExecutionPolicy Bypass -Scope Process
+   Import-Module Drive:\Path\to\VirtualMachineSgxSettings.psm1
+   Set-VMSgx -VmName MyVM -IsSgxEnabled $True -SgxSize 32
+   ```
+1. Start the VM and connect to it (right click, Connect...), finish the initial setup, reboot, and login.
+   - Enable OpenSSH server installation when given the choice during setup.
+   - All other options are sufficient to leave as the defaults or changed as desired.
+
+On the Linux build machine, or after opening an ssh session into the VM:
```

Comment:
Does VS have built-in ssh or should the user do this outside of VS?

Reply:
outside of VS.  Do you have a wording suggestion?

Reply:
I think the wording is fine. I was just curious if the VS support for Linux also included a builtin ssh terminal which would be cool.